hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -473,6 +473,10 @@ public void run() {
   // Cached clusterId on stand by masters to serve clusterID requests from clients.
   private final CachedClusterId cachedClusterId;
 
+  // Split/Merge Normalization plan executes asynchronously and the caller blocks on
+  // waiting max 5 sec for single plan to complete with success/failure.
+  private static final int NORMALIZATION_PLAN_WAIT_TIMEOUT = 5;
```

Comment:
nit: i like to include a unit in these types of constants. i.e., `NORMALIZATION_PLAN_WAIT_TIMEOUT_SEC`.

Reply:
I'm still not convinced we should do this, per my other comment :)